BannerlordTwitch/BannerlordTwitch/_Module/ModuleData/Languages/DE/strings-misc-DE.xml

```diff
@@ -452,111 +552,129 @@
         <!-- Okay -->
         <string id="hpFXglKx" text="Okay" original="Okay"/>
         <!-- Bannerlord Twitch Mod DISABLED -->
-        <string id="Sphd7XTS" text="Bannerlord Twitch Mod DEAKTIVIERT" original="Bannerlord Twitch Mod DISABLED"/>
+        <string id="Sphd7XTS" text="Bannerlord Twitch Mod DEAKTIVIERT"
+            original="Bannerlord Twitch Mod DISABLED"/>
         <!-- Bannerlord Twitch Mod STARTUP ERROR -->
-        <string id="cuXwwHRe" text="Bannerlord Twitch Mod STARTFEHLER" original="Bannerlord Twitch Mod STARTUP ERROR"/>
+        <string id="cuXwwHRe" text="Bannerlord Twitch Mod STARTFEHLER"
+            original="Bannerlord Twitch Mod STARTUP ERROR"/>
         <!-- Error in {Location}, please report this on the discord: {Error} -->
-        <string id="C0G8s2Lv" text="Fehler in {Location}, please report this on the discord: {Error}" original="Error in {Location}, please report this on the discord: {Error}"/>
+        <string id="C0G8s2Lv"
+            text="Fehler in {Location}. Bitte melde diesen Fehler in Discord: {Error}"
+            original="Error in {Location}, please report this on the discord: {Error}"/>
         <!-- SkillGroup -->
         <!-- None -->
-        <string id="p2ImRezI" text="Keiner" original="None"/>
+        <string id="p2ImRezI" text="Keine" original="None"/>
         <!-- All -->
         <string id="ZJUJQl8V" text="Alle" original="All"/>
         <!-- Melee -->
         <string id="Mtc9BiNI" text="Nahkampf" original="Melee"/>
         <!-- One Handed -->
         <string id="NRNrLcBt" text="Einhändig" original="One Handed"/>
         <!-- Two Handed -->
-        <string id="93kpWTwA" text="Beidhändig" original="Two Handed"/>
+        <string id="93kpWTwA" text="Zweihändig" original="Two Handed"/>
         <!-- Polearm -->
         <string id="PWibGbJ4" text="Stangenwaffe" original="Polearm"/>
         <!-- Ranged -->
         <string id="oidnJJjF" text="Fernkampf" original="Ranged"/>
         <!-- Bow -->
-        <string id="HW64G8xr" text="Sich beugen" original="Bow"/>
+        <string id="HW64G8xr" text="Bogen" original="Bow"/>
         <!-- Throwing -->
-        <string id="yR8vmGlW" text="Werfen" original="Throwing"/>
+        <string id="yR8vmGlW" text="Wurf" original="Throwing"/>
         <!-- Crossbow -->
         <string id="UG5KpbT7" text="Armbrust" original="Crossbow"/>
         <!-- Movement -->
         <string id="tVQs3vuO" text="Bewegung" original="Movement"/>
         <!-- Riding -->
-        <string id="14Unsmk1" text="Reiten" original="Riding"/>
+        <string id="14Unsmk1" text="Reitkunst" original="Riding"/>
         <!-- Athletics -->
-        <string id="4zlH32Z5" text="Leichtathletik" original="Athletics"/>
+        <string id="4zlH32Z5" text="Athletik" original="Athletics"/>
         <!-- Support -->
         <string id="AZD2nxRg" text="Unterstützung" original="Support"/>
```

Comment:
Is `Support` the proper word? In-game it's `Smithing` after Athletics

![image](https://user-images.githubusercontent.com/20150243/210069950-f186e3ee-ed4e-46d5-86b6-d39461c08301.png)

Reply:
Support is a set of skils, it includes: Scouting Trade Steward Medicine Engineering